app/AcquireIntersonArrayRF.cxx

```diff
@@ -217,10 +182,18 @@ int main( int argc, char * argv[] )
   Sleep( 100 ); // "time to stop"
   container.DisposeScan();
 
-  typedef itk::ImageFileWriter< ImageType > WriterType;
+  //
+  typedef itk::Image<short> WriterImageType;
+  typedef itk::CastImageFilter<ImageType, WriterImageType> CasterType;
```

Comment:
Agreed.  Avoid conversion on I/O...but this is a demo program, so not critical but good to set a good example.